### PR TITLE
Jarry/export psm

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

File: typescript/phoenix-sdk/src/utils/seatManager.ts [hunk 1]
```diff
@@ -181,23 +178,49 @@ export async function confirmOrCreateClaimSeatIxs(
...
 }
 
+/**
+ * Checks if the given trader has a seat on the given market
+ * If not, return claim seat instructions
+ * @param connection An instance of the Connection class
+ * @param market The market object
+ * @param trader The trader's address
+ */
+export async function createClaimSeatIdempotentInstructions(
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/utils/seatManager.ts:206]: i thought we wanted the idempotent instruction functions to not be async? if they are async, why don't we just check everything up front?

Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/utils/seatManager.ts:206]: I don't think we can make this one sync? I can try playing around with it a bit

Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/utils/seatManager.ts:206]: We can make it sync, but it's a decent amount of work to pass in the seat manager struct to get the DMM list. I think we should punt

Comment (eugene-chen) [reply typescript/phoenix-sdk/src/utils/seatManager.ts:206]: I don't think we can do it either but then I think we shouldn't call this idempotent

Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/utils/seatManager.ts]: fixed

**Review by eugene-chen — APPROVED** [review]

**Pull request merged**